app/models/katello/glue/pulp/pulp_content_unit.rb

```diff
@@ -36,7 +36,7 @@ def in_repositories(repos)
       end
 
       def pulp_data(uuid)
-        unit_handler.find_by(:unit_id => uuid)
+        unit_handler.find_by_unit_id(uuid)
```

Comment:
@komidore64 I don't think its really affecting anything, IMO just leave it. @ehelms any opinion?